### PR TITLE
Adds metric utils to convert Stats data models to Metric data models

### PR DESCRIPTION
**Pull request by isaikevych**
[Issue #190](https://github.com/census-instrumentation/opencensus-node/issues/190)

### CODE REVIEW

File: packages/opencensus-core/src/stats/metric-utils.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          return MetricDescriptorType.GAUGE_DOUBLE;
+        default:
+          throw new Error(`Unknown measure type ${measure.type}`);
+      }
+    }
+    throw new Error(`Unknown aggregation type ${aggregation}`);
+  }
+
+  /**
+   * Gets a MetricDescriptor for given view data.
```
[the remark this answers is not in the document]
Comment (isaikevych) [reply packages/opencensus-core/src/stats/metric-utils.ts]: Done

File: packages/opencensus-core/src/stats/metric-utils.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+   * @param view The view data to for which to build a metric descriptor
```
[the remark this answers is not in the document]
Comment (isaikevych) [reply packages/opencensus-core/src/stats/metric-utils.ts]: Done

File: packages/opencensus-core/src/stats/metric-utils.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      name: view.name,
+      description: view.description,
+      unit: view.measure.unit,
+      type: MetricUtils.getType(view.measure, view.aggregation),
+      labelKeys: view.getColumns().map(tag => ({key: tag} as LabelKey))
+    };
+  }
+
+  /**
+   * Converts tags to label values.
```
[the remark this answers is not in the document]
Comment (isaikevych) [reply packages/opencensus-core/src/stats/metric-utils.ts]: Done

File: packages/opencensus-core/src/metrics/export/types.ts [hunk 4]
```diff
@@ -135,16 +135,8 @@ export interface TimeSeries {
...
-  /** The value for the label */
-  readonly value: string;
-  /**
-   * If false the value field is ignored and considered not set.
-   * This is used to differentiate a missing label from an empty string.
-   */
-  readonly hasValue: boolean;
-}
+/** The LabelValue type. null value indicates an unset. */
+export type LabelValue = string|null;
```
Comment (OsvaldoRosado) [inline packages/opencensus-core/src/metrics/export/types.ts]: Is `|null` necessary? It looks like this project has `strictNullChecks` set to `false` in `tsconfig.json`

Comment (isaikevych) [reply packages/opencensus-core/src/metrics/export/types.ts]: We have plans to enable it in the future

**Pull request merged**